publication-commons/src/main/java/no/unit/nva/publication/doi/dto/Publication.java

```diff
@@ -0,0 +1,132 @@
+package no.unit.nva.publication.doi.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.net.URI;
+import java.util.List;
+import nva.commons.utils.JacocoGenerated;
+
+public class Publication {
+
```

Comment:
I think that the items that we are exchanging should be JSON-LD and not simple JSONs. Talk with @brinxmat about this.

Reply:
I had a session with @brinxmat yesterday, he said it wasnt needed for this, only for our APIs. 
You could say that messages on event bus is part of an API tho, even tho its for internal communication. 

I don't mind adding it as long as someone guides me thru this the first time, but @brinxmat didnt/ have the time yesterday.